app/scripts/modules/core/src/artifact/expectedArtifact.service.spec.ts

```diff
@@ -87,15 +88,15 @@ describe('ExpectedArtifactService', () => {
         id: 'artifact-id',
       };
       const kindConfig = ExpectedArtifactService.getKindConfig(artifact, false);
-      expect(kindConfig).toEqual(defaultKindConfig);
+      expect(kindConfig).toEqual(customKindConfig);
     });
 
     it('returns the default kind if neither kind nor type are stored on artifact when isDefault is true', () => {
       const artifact: IArtifact = {
         id: 'artifact-id',
       };
       const kindConfig = ExpectedArtifactService.getKindConfig(artifact, true);
-      expect(kindConfig).toEqual(defaultKindConfig);
+      expect(kindConfig).toEqual(customKindConfig);
     });
   });
 });
```

Comment:
Would it be worth adding a test ensuring that an artifact with no `kind` field but a `customKind` flag w/ a known type correctly results in `getKindConfig` returning the `customKindConfig`? E.g. something like:

```js
it('returns the default kind if an artifact has a customKind flag, regardless of its type', () => {
  const artifact: IArtifact = {
    id: 'artifact-id',
    customKind: true,
    type: 'foo-type'
  };
  const kindConfig = ExpectedArtifactService.getKindConfig(artifact, false);
  expect(kindConfig).toEqual(customKindConfig);
});
```

Reply:
Yes, good point; I'll add such a test